whimsylib/say.py

```diff
@@ -57,18 +57,35 @@ def _hear_voices(text, insanity):
     return "".join(segments)
 
 
+def output(msg):
+    """Print a message.
+
+    Unlike print(), this deals with de-denting and wrapping of text to fit
+    within the width of the terminal.
+
+    Paragraphs separated by blank lines in the input will be wrapped
+    separately.
+    """
+    msg = str(msg)
```

Comment:
Actually, since this is python3 I don't think we need the explicit string conversion.